duet/duet.model.lkml

```diff
@@ -12,3 +24,31 @@ explore: releases {
     ]
   }
 }
+
+##########################################################################################
+# below are explores defined for the MR1 dashboard
+# please hide these explores, as they've been developed specifially for the MR1 dashboard
+# and haven't been developed for general consumption
+explore: version_uplift {
+  from: version_uplift
```

Comment:
Here and others, we don't need `from` if the explore name matches the view name

Reply:
removed lines